bloom-instance/main.tf

```diff
@@ -15,8 +15,6 @@ terraform {
 
 provider "aws" {
   region = var.aws_region
-
-  default_tags = local.default_tags
```

Comment:
context: This line being here was erroring out when i tried `terraform plan`ing. Looking at the documentation (https://registry.terraform.io/providers/hashicorp/aws/latest/docs#aws-configuration-reference) default_tags is not a valid argument so I think this is just a cruft/bug and removed it.

Reply:
The usage of the default_tags arg was actually incorrect, though I missed it in the back and forth during review.  The correct usage looks like this:

`
provider "aws" {
  ...
  default_tags {
    tags = local.default_tags
  }
}
`

Note that we also don't need local.default tags to be defined (we can just put our tags right into the provider block), but it really doesn't hurt anything either.

Reply:
Hmm tags is not a supported argument either, I think that many of the services (e.g. the VPC) do support a tags arg though. I'll leave this out and continue to put the tags into the VPC directly?

![image](https://user-images.githubusercontent.com/10789523/220432343-c71f1afe-d00c-4516-9934-3080ad59a645.png)

Reply:
Also I used default_tags_with_name, not default_tags on the vpc service -- lmk which you think is better

Reply:
You can think of the default_tags arg as an object with a single property named "tags" that is basically a map<string>.  The code snippet I posted above has the proper syntax (if oddly formatted), but it basically translates to `default_tags.tags = local.default_tags`.

We do need a Name tag for the VPC though, so go ahead and use default_tags_with_name.

Reply:
ooooohh!!! OK thanks for the clarification got it. I should have looked more closely at your code snippet, my bad. Added that which doesn't throw an error when I `terraform plan`. Resolving this now